### PR TITLE
removing dependencies

### PR DESCRIPTION
**Pull request by julien51**
# Description

Removed some unused dependencies at the root and inside unlock-app.

# Issues

Refs #4510

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: tickets/package.json [hunk 1]
```diff
@@ -55,10 +55,9 @@
     "react-redux": "7.1.0",
     "react-router": "5.0.1",
     "react-select": "3.0.4",
-    "react-test-renderer": "16.8.6",
+    "react-test-renderer": "^16.9.0",
```
Comment (julien51) [inline tickets/package.json]: This is strange!

**Comment by cnasc** [comment]
Very weird integration test failure on this.

**Comment by julien51** [comment]
yup. rereunning now!

**Comment by julien51** [comment]
Interesting. it keeps failing with the same error.

**Comment by julien51** [comment]
Very strange. code coverage went down because I removed some dependencies.

**Comment by cnasc** [comment]
Coverage went down because a test failed to run:

```
FAIL src/__tests__/storybook/storyshots.test.js
  ● Test suite failed to run

    Cannot find module 'react-test-renderer' from 'renderTree.js'

      at Resolver.resolveModule (../node_modules/jest-resolve/build/index.js:259:17)
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/frameworks/react/renderTree.js:8:49)
```

**Comment by julien51** [comment]
Alright, added the dependency again!

**Pull request merged**